src/pykiso/lib/auxiliaries/communication_auxiliary.py

```diff
@@ -61,8 +61,15 @@ def receive_message(
         log.debug(
             f"retrieving message in {self} (blocking={blocking}, timeout={timeout_in_s})"
         )
-        msg = self.wait_and_get_report(blocking=blocking, timeout_in_s=timeout_in_s)
-        log.debug(f"retrieved message '{msg}' in {self}")
+        response = self.wait_and_get_report(
+            blocking=blocking, timeout_in_s=timeout_in_s
+        )
+        log.debug(f"retrieved message '{response}' in {self}")
+        msg = response.get("msg")
+        remote_id = response.get("remote_id")
```

Comment:
I decide to not break the com_aux return type because I don't really see any benefit based on how this auxiliary is used (maybe I missed something regarding this auxiliary).